school-library/students/student.rb

```diff
@@ -0,0 +1,12 @@
+require '../person/persons'
+
+class Student < Animal
+  def initialize(classroom)
+    super(name, 21, true)
```

Comment:
- [ ] The parameters here should be the parameters given in the `Person` class.e.g. `super(age, name, parent_permission)`. Kindly fix this.